### ISSUE TITLE
A flag for ViewfinderView, to hide the red line, and the yellow points

### ISSUE THREAD

**Issue by kangle1208** [journeyapps/zxing-android-embedded#263]
Hi rkistner
Thank you very much for yours "zxing-android-embedded"

When scanning code, there is a red line, and some yellow points, on the screen

If users don't want to see the red line and yellow points
they need to modify the source, and rebuild a jar or aar file
and then, import the jar/aar file into their project
and use it like this: 
compile( name: 'zxing-android-embedded-3.4.0', ext: 'aar' )

If add a flag for ViewfinderView.java, and do some change in the "onDraw" method
Users just set the flag, and no need to build jar/aar file
Just use it like this: 
compile 'com.journeyapps:zxing-android-embedded:3.4.0'

Just for more simple, is this usefull?

Thank you very much, again!

**Comment by rkistner** [comment]
Yes, that could be a useful shortcut. Pull requests welcome.

**Comment by chris8447** [comment]
Yeah please change it so that it is possible to set it via the Java class.

Great work guys! 😊

**Comment by AbubakarSiddique** [comment]
Is it implemented?

**Comment by jawed1412** [comment]
just put in you colors.xml 
"
< color name="zxing_viewfinder_laser" >#00ffffff</color>
< color name="zxing_possible_result_points" >#00ffffff</color>
"
